Language/Generics/Superbounded_types/well_bounded_A06_t02.dart

```diff
@@ -2,13 +2,11 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion It is a compile-time error if a parameterized type `T` is
-/// super-bounded when it is used in any of the following ways:
-/// - `T` is an immediate subterm of a new expression or a constant object
-///   expression.
+/// @assertion Any use of a type `T` which is not well-bounded is a
+/// compile-time error.
 ///
-/// @description Checks that it is a compile-time error if a parametrized
-/// super-bounded type is used in a constant `is` expression and not a top type.
+/// @description Checks that it is a compile-time error if not a well-bounded
+/// type is used as a right hand operand in an `is` expression.
```

Comment:
Nit.
```suggestion
/// type is used as the right hand operand in an `is` expression.
```
We always know _which_ right hand operand it is, because there is defi-nit-ely only one. ;-)